src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -109,4 +110,13 @@ public ResponseEntity<List<Hearing>> casesCaseIdHearingsGet(
         return new ResponseEntity<>(caseService.getCaseHearings(caseId), HttpStatus.OK);
     }
 
+    @Override
+    public ResponseEntity<SingleCase> casesCaseIdGet(
+        @Parameter(name = "caseId", description = "caseId is the internal cas_id of the case.", required = true, in = ParameterIn.PATH) @PathVariable("caseId") Integer caseId
```

Comment:
Do we need to repeat these annotations/constraints from the interface, will they not be inherited?

Reply:
yes, I believe so, but all other methods have these annotations - so should I update each method and remove the annotations?